src/lib/redux/inventoryDataSlice.ts

```diff
@@ -2,19 +2,55 @@
 /* eslint-disable no-param-reassign */
 import { createEntityAdapter, createSlice, EntityState } from '@reduxjs/toolkit';
 import { InventoryRecord, InventoryUpdateRecord, ProductRecord, PurchaseRequestRecord } from '../airtable/interface';
+import { setCurrSite } from './siteDataSlice';
+import { RootState } from './store';
 
 const inventoryUpdatesAdapter = createEntityAdapter<InventoryUpdateRecord>();
+const productsAdapter = createEntityAdapter<ProductRecord>();
+const siteInventoryAdapter = createEntityAdapter<InventoryRecord>();
+const purchaseRequestsAdapter = createEntityAdapter<PurchaseRequestRecord>();
```

Comment:
This entire setup and file looks really nice, really good work :)

I'll work on refactoring `customerData` and `siteData` to look similar to this